torch/fx/subgraph_rewriter.py

```diff
@@ -189,6 +215,10 @@ def forward(self, x, w1, w2):
                               remove_overlapping_matches=True)
     _matches: List[InternalMatch] = matcher.match(original_graph)
 
+    # Filter out matches that don't match the filter
+    if match_filter:
+        _matches = [m for m in _matches if match_filter(m, original_graph, pattern_graph)]
```

Comment:
do we need `original_graph`? I don't know how `original_graph` will be helpful in filtering the `match` since it won't know where to look

Reply:
I want to keep the function signature general, so that I won't need to come back to this and add another field. 
My guess is that original_graph might be useful for filtering out the matches, if it need to check the surrounding nodes of the match.